test_hdf5_cve.sh

```diff
@@ -70,13 +70,31 @@ fi
 H5DUMP=$bindir/h5dump
 H5CLEAR=$bindir/h5clear
 H5DEBUG=$bindir/h5debug
-GIF2H5=$bindir/gif2h5
 H5FORMAT_CONVERT=$bindir/h5format_convert
 H5LS=$bindir/h5ls
-H52GIF=$bindir/h52gif
```

Comment:
Do these ever get set below? They're used later on in the script but it doesn't look like they get set. It should be fine to always set them here, even when the tool is missing.